app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt

```diff
@@ -78,9 +78,13 @@ fun Context.hideKeyboard(view: View) {
     inputMethodManager.hideSoftInputFromWindow(view.windowToken, 0)
 }
 
-@RequiresApi(Build.VERSION_CODES.R)
-suspend fun Context.hideKeyboardIfVisible(view: View, inputView: View, delayMillis: Long = 0) {
-    if (view.rootWindowInsets?.isVisible(WindowInsets.Type.ime()) == true) {
+suspend fun Context.hideKeyboardWithDelay(inputView: View, delayMillis: Long = 0) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+        if (inputView.rootView.rootWindowInsets?.isVisible(WindowInsets.Type.ime()) == true) {
```

Comment:
I think there's actually a compat-variant of this API! We should check that out, in that case we could move to only having one code path

Reply:
Investigating!

Reply:
We need `androidx.core:core-ktx:1.5.0-beta03` In order to access `WindowInsetsCompat.Type`. Should we upgrade or wait for stable?

Reply:
I'm ok with upgrading 🙂

Reply:
Done!